src/PlusCommon/PlusCommon.cxx

```diff
@@ -319,20 +319,22 @@ PlusStatus PlusCommon::CreateTemporaryFilename(std::string& aString, const std::
 std::vector<std::string> PlusCommon::GetSequenceExtensions()
 {
   // Return list of supported sequence file extensions in lower case
-  std::vector<std::string> sequenceExtensions;
-  sequenceExtensions.push_back(".igs.mha");
-  sequenceExtensions.push_back(".igs.mhd");
-  sequenceExtensions.push_back(".igs.nrrd");
-  sequenceExtensions.push_back(".igs.nhdr");
-  sequenceExtensions.push_back(".seq.mha");
-  sequenceExtensions.push_back(".seq.mhd");
-  sequenceExtensions.push_back(".seq.nrrd");
-  sequenceExtensions.push_back(".seq.nhdr");
-  sequenceExtensions.push_back(".mha");
-  sequenceExtensions.push_back(".mhd");
-  sequenceExtensions.push_back(".nrrd");
-  sequenceExtensions.push_back(".nhdr");
-  return sequenceExtensions;
+  return std::vector<std::string>(
+  {
+    ".igs.mha",

```

Comment:
Can you revert this section please?
I had to make the change in https://github.com/PlusToolkit/PlusLib/commit/d285b9a94b7b1d5a7b89912e5a27c05cdbd70a23 since it wouldn't compile on C++98.